Formula/clojure-lsp.rb

```diff
@@ -27,16 +27,15 @@ def install
   end
 
   test do
-    require "Open3"
+    ENV["JAVA_HOME"] = Formula["openjdk@11"].opt_prefix
```

Comment:
This shouldn't be needed. We need to write `JAVA_HOME` to the `write_jar_script` in that case